Gems/PhysX/Code/Tests/PhysXMeshColliderComponentModeTests.cpp

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <TestMeshColliderComponent.h>
+
+#include <AzManipulatorTestFramework/AzManipulatorTestFrameworkTestHelpers.h>
+#include <AzToolsFramework/UnitTest/AzToolsFrameworkTestHelpers.h>
+#include <AzToolsFramework/ViewportSelection/EditorSelectionUtil.h>
+#include <AzToolsFramework/Entity/EditorEntityHelpers.h>
+
+namespace UnitTest
+{
+    class PhysXMeshColliderComponentModeTest
+        : public ToolsApplicationFixture<false>
+    {
+    protected:
+        using EntityPtr = AZ::Entity*;
+
+        AZ::ComponentId m_meshColliderComponentId;
+
+        EntityPtr CreateMeshColliderComponent()
+        {
+            AZ::Entity* entity = nullptr;
+            AZ::EntityId entityId = CreateDefaultEditorEntity("ComponentModeEntity", &entity);
+
+            entity->Deactivate();
+
+            // Add placeholder component which implements component mode.
+            auto meshColliderComponent = entity->CreateComponent<TestMeshColliderComponent>();
+
+            m_meshColliderComponentId = meshColliderComponent->GetId();
+
+            entity->Activate();
+
+            AzToolsFramework::SelectEntity(entityId);
+
+            return entity;
+        }
+
+        // Needed to support ViewportUi request calls.
+        ViewportManagerWrapper m_viewportManagerWrapper;
+
+        void SetUpEditorFixtureImpl() override
+        {
+            m_viewportManagerWrapper.Create();
+        }
+        void TearDownEditorFixtureImpl() override
+        {
+            m_viewportManagerWrapper.Destroy();
+        }
+    };
+
+    TEST_F(PhysXMeshColliderComponentModeTest, PressingKeyRShouldResetAssetScale)
+    {
+        // Given there is a sphere collider component in component mode.
```

Comment:
Looks like a copy-paste error in this comment

Reply:
It came from `PhysXColliderComponentModeTest`. I'll remove it in both.